bin/container.sh

```diff
@@ -21,7 +21,6 @@ mysql=mysql:5.6
 docker_network=${project}_nw
```

Comment:
During the merge I will make sure to use the `bin` dir on develop, thanks for pointing this out!